src/polkitmateauthenticator.c

```diff
@@ -308,8 +308,11 @@ session_request (PolkitAgentSession *session,
     }
 
   gtk_widget_show_all (GTK_WIDGET (authenticator->dialog));
-  gtk_window_present_with_time (GTK_WINDOW (authenticator->dialog),
-                                gdk_x11_get_server_time (gtk_widget_get_window (GTK_WIDGET (authenticator->dialog))));
+  if (GDK_IS_X11_WINDOW (gtk_widget_get_window (GTK_WIDGET (authenticator->dialog))))
+    gtk_window_present_with_time (GTK_WINDOW (authenticator->dialog),
+                                  gdk_x11_get_server_time (gtk_widget_get_window (GTK_WIDGET (authenticator->dialog))));
+  else
+    gtk_window_present (GTK_WINDOW (authenticator->dialog));
   password = polkit_mate_authentication_dialog_run_until_response_for_prompt (POLKIT_MATE_AUTHENTICATION_DIALOG (authenticator->dialog),
```

Comment:
Indent of line 315 doesn't match lines below.

Reply:
What do you mean?

Reply:
Indent of line 315 doesn't match the indent of lines below.

Reply:
Line 315 is in an else case? Do you mean using with curly brackets? What changes do I need to make

Reply:
Are line 316-320 not in else case? (password........)
If yes, sync line 315 with 316-320.
Still wondering that you don't see that code-style issue?
```